contracts/lupi.sol

```diff
@@ -15,17 +20,35 @@ contract lupi is owned {
     State state = State.Betting;
 
     Ticket[] tickets;
-    uint revealedCount;
+    uint revealedCount; // TODO: shall we replace this with a getter returning revealedBets.length?
```

Comment:
No, it would be a bug. The revealedCount is the number of all reveals. Lenght is number of uniques.